test-reports/results.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite errors="0" failures="0" hostname="Qiaos-MBP.lan" name="pytest" skipped="0" tests="14" time="6.250" timestamp="2020-04-30T12:50:32.057845"><testcase classname="tests.test_es_models" file="tests/test_es_models.py" line="9" name="test_missing_required_args[input_args0]" time="0.003"></testcase><testcase classname="tests.test_es_models" file="tests/test_es_models.py" line="9" name="test_missing_required_args[input_args1]" time="0.002"></testcase><testcase classname="tests.test_es_models" file="tests/test_es_models.py" line="9" name="test_missing_required_args[input_args2]" time="0.002"></testcase><testcase classname="tests.test_es_models" file="tests/test_es_models.py" line="9" name="test_missing_required_args[input_args3]" time="0.002"></testcase><testcase classname="tests.test_es_models" file="tests/test_es_models.py" line="9" name="test_missing_required_args[input_args4]" time="0.001"></testcase><testcase classname="tests.test_es_models" file="tests/test_es_models.py" line="9" name="test_missing_required_args[input_args5]" time="0.002"></testcase><testcase classname="tests.test_es_models" file="tests/test_es_models.py" line="25" name="test_get_es_models_standard_behavior" time="5.938"></testcase><testcase classname="tests.test_es_models" file="tests/test_es_models.py" line="43" name="test_get_es_models_parametrized[mock_listdir0]" time="0.001"></testcase><testcase classname="tests.test_es_models" file="tests/test_es_models.py" line="43" name="test_get_es_models_parametrized[mock_listdir1]" time="0.001"></testcase><testcase classname="tests.test_es_models" file="tests/test_es_models.py" line="57" name="test_get_es_models_from_directory" time="0.021"></testcase><testcase classname="tests.test_es_models" file="tests/test_es_models.py" line="70" name="test_get_models_with_descriptions" time="0.007"></testcase><testcase classname="tests.test_es_models" file="tests/test_es_models.py" line="80" name="test_get_models_no_descriptions" time="0.005"></testcase><testcase classname="tests.test_es_models" file="tests/test_es_models.py" line="90" name="test_get_models_empty_descriptions" time="0.005"></testcase><testcase classname="tests.test_es_models" file="tests/test_es_models.py" line="100" name="test_get_models_descriptions_with_other_data" time="0.006"></testcase></testsuite></testsuites>
```

Comment:
I don't think we need this.

Reply:
Yes, this file should be omitted.

Reply:
Should I add test-reposts/ to .gitignore?